libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4853,35 +4853,41 @@ void GCS_MAVLINK::send_water_depth() const
     }
 
     RangeFinder *rangefinder = RangeFinder::get_singleton();
-    if (rangefinder == nullptr || !rangefinder->has_data_orient(ROTATION_PITCH_270)) {
-        // no rangefinder or not facing downwards
-        return;
-    }
 
-    const bool sensor_healthy = (rangefinder->status_orient(ROTATION_PITCH_270) == RangeFinder::Status::Good);
+    for (uint8_t i=0; i<RANGEFINDER_MAX_INSTANCES; i++) {
+        const AP_RangeFinder_Backend *s = rangefinder->get_backend(i);
+        
+        if (s == nullptr || s->orientation() != ROTATION_PITCH_270 || !s->has_data()) {
+            continue;
+        }
 
-    // get position
-    const AP_AHRS &ahrs = AP::ahrs();
-    Location loc;
-    IGNORE_RETURN(ahrs.get_position(loc));
+        // get position
+        const AP_AHRS &ahrs = AP::ahrs();
+        Location loc;
+        IGNORE_RETURN(ahrs.get_position(loc));
```

Comment:
I'm not sure if we need to get the location at the moment each backend's data is retrieved? I don't imagine going through the backends would take enough time to matter? Either way it should match with the Log function.

@rmackay9 your opinion?

Reply:
yes, I would probably pull the get_position() outside of the loop.. but then you might want a checker higher up that there is at least one lidar that is downward facing and exit early.  Sorry, maybe that's already there, I haven't checked this PR fully yet..

Reply:
Cool. Yeah the check for downward facing is higher up.

@shiv-tyagi Would you please the get position bit out of the loop? Then I think this one is good.

Reply:
Sure! Moved it outside the loop. Please check.